api/build.gradle

```diff
@@ -101,6 +105,13 @@ sourceSets {
     compileClasspath += generated.output
     runtimeClasspath += generated.output
   }
+  integration {
```

Comment:
When do you envision integration tests running? Do we want them to be a part of circle?

How long does this test take to run?

Reply:
Right now they only take a few seconds, but they do touch external stuff and could mess with our quotas and such. I could see them running nightly. For now, I'm fine with manual only.